bundles/org.openhab.binding.velbus/src/main/resources/OH-INF/config/config.xml

```diff
@@ -530,6 +530,11 @@
 			<label>Address</label>
 			<description>The velbus address of the device</description>
 		</parameter>
+		<parameter name="subaddress1" type="text" required="true">
+			<label>Sensor Address</label>
+			<description>The velbus sub-address of the device (FF = disabled)</description>
+			<default>FF</default>
```

Comment:
A couple of observations here. I would perhaps try to include the numeric channel number in the label, as Velbus Link does not separately call out the channel groups, and it is much easier to reason that you need to provide a sub-address for CH9.

```suggestion
			<label>Sensor Channel CH9 Address</label>
			<description>The velbus sub-address of the device (FF = disabled)</description>
			<default>FF</default>
```

Additionally, is `FF` truly the correct default to use here? I'm not sure what are the semantics of `required = "true"` together with `<default>` but if it simply prefills the field with `FF`, then it is going to be pretty poor experience for many people who have multi-address modules, as leaving this value as a default will result in things that can only expose partial functionality of the modules.

It would be much better if disabling these channel groups required a conscious action (i.e. user manually typing in `FF`,) whether in the UI or in the textual configs.

Reply:
It's the default value in VelbusLink when a module is manually added. 
The value is automatically changed with the one returned by the module during a scan.
People who add manually a module in OpenHAB should know what they make and will change the value to the right one.
I prefer to have a default value than an error.